script.js

```diff
@@ -42,9 +42,143 @@ contactLink.addEventListener('click', (e) => {
 });
 
 const workSection = document.getElementById('section-2');
-console.log(workSection);
-
 const works = document.querySelectorAll('.work');
-console.log(works);
+workSection.innerHTML = '';
+
+/**
+ * This class can return works objects
+ */
+class Work {
+  /**
+   * @param {string} id
+   * @param {string} imgSrc
+   * @param {string} name
+   * @param {string} smallDescription
+   * @param {string} summary
+   * @param {string} usedTools
+   *  */
+  constructor(id, imgSrc, name, smallDescription,
+      summary, usedTools) {
+    this.id = id;
+    this.imgSrc = imgSrc;
+    this.name = name;
+    this.smallDescription = smallDescription;
+    this.summary = summary;
+    this.usedTools = usedTools;
+  }
```

Comment:
- [ ] You did an amazing job adding the required JS to your popup :+1:, but as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) the JavaScript code should implement an array of objects to store JavaScript objects containing information like, name, description, featured image, technologies, link to live version, and link, for each project (Works section). You can see an example below.

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

Kindly don't forget that the modal popup and the works section data should be populated with the data from the `projects` array.